README.md

```diff
@@ -2,19 +2,105 @@
 
 A small set of abstractions that will help you quickly build an Express.js app that consumes the Shopify API.
 
-## Architecture
+## Example
```

Comment:
I love putting inline examples here. I was going to do this but now I don't have to :)